internal/pkg/deploy/cloudformation/stack/workload.go

```diff
@@ -186,19 +182,30 @@ func serializeTemplateConfig(parser template.Parser, stack templateConfigurer) (
 	if err != nil {
 		return "", err
 	}
-	doc, err := parser.Parse(wkldParamsTemplatePath, struct {
-		Parameters []*cloudformation.Parameter
-		Tags       []*cloudformation.Tag
+
+	tags := stack.Tags()
+
+	config := struct {
+		Parameters map[string]*string `json:"Parameters"`
+		Tags       map[string]*string `json:"Tags,omitempty"`
 	}{
-		Parameters: params,
-		Tags:       stack.Tags(),
-	}, template.WithFuncs(map[string]interface{}{
-		"inc": template.IncFunc,
-	}))
+		Parameters: make(map[string]*string, len(params)),
+		Tags:       make(map[string]*string, len(tags)),
+	}
+
+	for _, param := range params {
+		config.Parameters[aws.StringValue(param.ParameterKey)] = param.ParameterValue
+	}
+	for _, tag := range tags {
+		config.Tags[aws.StringValue(tag.Key)] = tag.Value
+	}
+
+	str, err := json.MarshalIndent(config, "", "  ")
 	if err != nil {
 		return "", err
 	}
-	return doc.String(), nil
+
+	return string(str), nil
```

Comment:
nit: should we wrap the err since it's an external call?
```go
out, err := json.MarshalIndent()
if err != nil {
   return "", fmt.Errorf("marshal stack parameters to JSON: %v", err)
}
return string(out), nil
```

Reply:
oh yeah, good idea